runtime/gc_vlhgc/ConfigurationIncrementalGenerational.cpp

```diff
@@ -136,6 +136,9 @@ MM_ConfigurationIncrementalGenerational::createHeapWithManager(MM_EnvironmentBas
 
 #if defined(J9VM_ENV_DATA64)
 	extensions->indexableObjectModel.setIsDataAddressPresent(true);
+	J9JavaVM *vm = (J9JavaVM *)extensions->getOmrVM()->_language_vm;
+	/* set indexableObjectLayout = J9IndexableObjectLayout_DataAddr_Arraylet (Balanced GC and off-heap disabled) */
```

Comment:
again, be a more creative, something like:

"Let VM know that indexable objects in Balanced always have dataAddr, and let's assume initially it has arraylets, which can be later overridden if Offheap is Enabled.